core/tests/protractor_desktop/ExplorationTranslationTab.js

```diff
@@ -136,6 +136,58 @@ describe('Exploration translation', function() {
     users.logout();
   });
 
+  it('should contain accessibility elements', function() {
+    users.createUser('user2@translationTab.com', 'user2TranslationTab');
+    users.login('user2@translationTab.com');
+    workflow.createExploration();
+
+    explorationEditorMainTab.setStateName('first');
```

Comment:
In #6340, @rakshitkumarcse is creating a common exploration for the translation tests. Could you coordinate with him, to make sure the same exploration is used here too? (For now I am happy to keep it as is, and when his PR is merged, make sure to change the test to use the same exploration).